api/src/queries/survey/survey-view-queries.ts

```diff
@@ -196,19 +196,49 @@ export const getSurveyFocalSpeciesDataForViewSQL = (surveyId: number): SQLStatem
   `;
 };
 
-export const getSurveyAncillarySpeciesDataForViewSQL = (surveyId: number): SQLStatement | null => {
+// export const getSurveyAncillarySpeciesDataForViewSQL = (surveyId: number): SQLStatement | null => {
```

Comment:
Deleting function?

Reply:
Yes, in the getSpeciesData function, I now get the species list once from the DB and parse them into focal and ancillary using the model.